services/src/main/java/org/exoplatform/news/queryBuilder/NewsQueryBuilder.java

```diff
@@ -19,6 +19,8 @@
 
   private static final String PUBLISHER_MEMBERSHIP_NAME       = "publisher";
 
+  private static final String EXO_SPACE_ID      = "exo:spaceId = '";
```

Comment:
```suggestion
  private static final String EXO_SPACE_ID      = "exo:spaceId";
```
you can keep " = '" in the query